Ports/frotz/patches/ReadMe.md

```diff
@@ -0,0 +1,7 @@
+# Patches for frotz on SerenityOS
+
+## `0001-Set-UNAME_S-to-SerenityOS-and-LINUX-to-yes.patch`
+
+Set `UNAME_S` to SerenityOS and `LINUX` to yes
```

Comment:
This patch description describes _what_ you changed , but you can easily see that by looking at the change itself.

Please update the patch description with the reason _why_ this change is needed: what exactly does it fix, and how?

Reply:
Sorry, but I cannot explain quirks in every port's build system. I'm not sure why this works, in fact I depend on testers to know if it works on other platforms.

Reply:
You don't necessarily need a deep understanding, but whatever reason you had for adding that patch, it would be helpful to write that in the patch description. In the future, we might not need the patch any more, but it's hard to tell if we don't know why it was added in the first place. :^)